config/overlays/odh/patches/manager_auth_config_patch.yaml

```diff
@@ -0,0 +1,26 @@
+# This patch injects auth config properties from auth-refs configmap in the controller manager
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        # name must match the volume name below
+        env:
+          - name: DEFAULT_AUTH_PROVIDER
+            valueFrom:
+              configMapKeyRef:
+                # AUTH_NAMESPACE has the external authorization provider name
+                # used in the istio servicemeshcontrolplane (smcp) config
+                key: AUTH_NAMESPACE
+                name: auth-refs
```

Comment:
where is `auth-refs`config defined? trying to see where are default values defined.

Reply:
`oc get cm -n opendatahub auth-refs -o yaml`

Reply:
I mean where is this configmap being created with default values, which I thought is source where the default values are supplied from

Reply:
I took a look at the odh-operator, it seems to be created there and takes the data from the `DSCInitialization` file to fill in the defaults.

refs:

https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/pkg/feature/servicemesh/resources.go#L45-86

Reply:
great! thank you for the reference.